src/main/java/me/voidxwalker/autoreset/mixin/config/CreateWorldScreenMixin.java

```diff
@@ -173,7 +174,7 @@ private void updateLevelNameField(boolean moreOptionsOpen, CallbackInfo ci) {
             cancellable = true
     )
     private void saveAtumConfigurations(CallbackInfo ci) {
-        if (!this.isAtum() || Atum.isRunning()) {
+        if (!isAtumConfig()) {
```

Comment:
This check would mean it saves on non-atum screens too, you can just replace it with
```
if (this.isAtumConfig()) {
    this.save();
    this.closeConfigScreen();
    ci.cancel();
}
```

Reply:
Yea I need sleep idk